### DISCUSSION TITLE
Any way to access the raw mongodb driver inside prisma

### DISCUSSION THREAD

**Discussion by shash7** [prisma/prisma#10899] [Q&A] [answered]
I want to set writeconcern on a specific query. Is there a way to access the raw mongodb driver to do this?

**Comment by janpio** [comment]
No, that is not possible. But setting the writeconcern per query might be a valid use case, can you please open a feature request for this and describe the problem you are trying to solve this way? Thanks.

**Reply by shash7** [reply]
That's alright. With the new raw mongodb access I can specify writeconcern, although it would be neater if I could specify it through prisma client.

**Reply by janpio** [reply]
Great. Feature request with a bit of explanation why specifying writeconern per Prisma Client query would be desirable still though. You are probably not alone with that need.

**Reply by shash7** [reply, marked as answer]
> Great. Feature request with a bit of explanation why specifying writeconern per Prisma Client query would be desirable still though. You are probably not alone with that need.

#11939

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
